### PR TITLE
Feature/arc 401

### PR DESCRIPTION
**Pull request by ian-emsens-shd**
https://meemoo.atlassian.net/browse/ARC-401
https://meemoo.atlassian.net/browse/ARC-402

### CODE REVIEW

File: src/modules/cp/const/requests.const.ts [hunk 1]
```diff
@@ -53,5 +55,16 @@ export const requestStatusFilters = (): TabProps[] => {
 };
 
 export const requestCreatedAtFormatter = (date: Date): string => {
```
Comment (bertyhell) [inline src/modules/cp/const/requests.const.ts:57]: is this date the createdAt date?
then consider naming it: createdAtDate

**Review by bertyhell — APPROVED** [review]

File: src/modules/cp/const/requests.const.ts (hunk 1 above)
Comment (ian-emsens-shd) [reply src/modules/cp/const/requests.const.ts:57]: This is in line with other constants right now but in general, prefixing these small functions with their context-object is desirable.

Going forward I expect the following;

`src/modules/shared/const/dates.ts`
```
const formatDateToFivePreviousDays = (...) => {...}
```

`src/modules/cp/const/requests.const.ts`
```
const requestCreatedAtFormatter = formatDateToFivePreviousDays;
```

`src/modules/some/other/module/const/foobar.const.ts`
```
const foobarPropertyFormatter = formatDateToFivePreviousDays;
```

File: src/modules/cp/components/ApproveRequestBlade/ApproveRequestBlade.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+					)}
+				</strong>
+				<p>{props.selected?.time}</p>
+			</div>
+
+			<div className="u-px-32">
+				<FormControl
+					className={clsx(styles['c-approve-request-blade__date-time'], 'u-mb-32')}
+					errors={[errors.accessFrom?.message]}
+					label={t('Van')}
```
[the remark this answers is not in the document]
Comment (ian-emsens-shd) [reply src/modules/cp/components/ApproveRequestBlade/ApproveRequestBlade.tsx:130]: Resolved in #119

File: src/modules/cp/const/requests.const.ts (hunk 1 above)
Comment (andrych) [reply src/modules/cp/const/requests.const.ts:57]: Personally `requestCreatedAtDateFormatter` seems more descriptive, it lets you know the formatter is intended to be used on dates.

**Pull request merged**